### ISSUE TITLE
Change country dinamically

### ISSUE THREAD

**Issue by sombatos** [LouisMazel/vue-phone-number-input#30]
Is there any way to change selected country after the component has been displayed?

I have `default-country-code` set to:
```
:default-country-code="phone_country"
```
When changing `phone_country` selected country is not changing in the form.

**Comment by sombatos** [comment]
Solution:
```
<template>
    <vue-phone-number-input v-model="inputs.phone_number" ref="phone_number" />
</template>

<script>
    var phone_code = 'US';
    // Remember phone number because setting country code clears the phone number
    var phone_backup = this.inputs.phone_number;
    this.$refs.phone_number.countryCode = phone_code;
    // Restore the phone number
    this.$refs.phone_number.phoneNumber = phone_backup;
</script>
```

**Comment by LouisMazel** [comment]
I will fix it today, more infos here: https://github.com/LouisMazel/vue-phone-number-input/issues/27

**Comment by JRB-y** [comment]
I think it's not fixed yet !

**Comment by LouisMazel** [comment]
Solved in 1.1.7

Release : https://github.com/LouisMazel/vue-phone-number-input/releases/tag/1.1.7
